src/include/sof/schedule/ll_schedule_domain.h

```diff
@@ -43,6 +43,7 @@ struct ll_schedule_domain_ops {
 
```

Comment:
s/letancy/latency in commit message.

Also, care to enhance the commit message as to make very clear *why* do we need this change. What scenario did you run and with this commit makes things better.

Since this is a sensible topic I want to better understand the reasoning behind this patch.